index.html

```diff
@@ -5,17 +5,64 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="This is the project that demonstrates the learners understanding of documenting professionally as well as using linters">
-    <title>Hello Microverse</title>
+    <meta name="description"
+        content="This is the description of william' portfolio">
+    <title>William | Portfolio</title>
     <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
+        rel="stylesheet">
 </head>
 
 <body>
     <link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
     <noscript>
         <link rel="stylesheet" href="styles.css">
     </noscript>
-    <h1 class="header">Hello Microverse!</h1>
+    <div class="wrap">
+        <header class="toolbar">
+            <a href="#">
+                <h4>My Logo</h4>
```

Comment:
[OPTIONAL]Great job implementing the requested change. You could remove the `h4` entirely and just have it as `<a href="#">My Logo </a>.` It makes more sense using elements when you really need them